modules/ROOT/nav.adoc

```diff
@@ -161,9 +161,10 @@
 *** xref:manage:security/iam-roles.adoc[]
 ** xref:manage:tiered-storage-linux/index.adoc[Tiered Storage]
 *** xref:manage:tiered-storage.adoc[Tiered Storage]
```

Comment:
Should this page be labelled differently to avoid having two pages with the same name.